doc/source/whatsnew/v0.23.1.txt

```diff
@@ -10,12 +10,70 @@ and bug fixes. We recommend that all users upgrade to this version.
     :local:
     :backlinks: none
 
+.. _whatsnew_0231.enhancements:
 
 .. _whatsnew_0231.fixed_regressions:
 
 Fixed Regressions
 ~~~~~~~~~~~~~~~~~
 
+**Comparing Series with datetime.date**
+
+We've reverted a 0.23.0 change to comparing a :class:`Series` holding datetimes and a ``datetime.date`` object (:issue:`21152`).
```

Comment:
use ``date=datetime64[ns]``

Reply:
I'm not sure what you mean here.

Reply:
oh was just saying that users talk about dtypes, rather than ``Series`` holding datetimes. no big deal really.